numpy/core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -1724,10 +1738,20 @@ mapiter_@name@(PyArrayMapIterObject *mit)
                         NPY_AUXDATA_FREE(transferdata);
                         return -1;
                     }
+                    if (is_reset_trivial != 0) {
+                        /* resetting is trivial, calculate the offsets */
+                        reset_offsets[0] = subspace_ptrs[0] - self_ptr;
+                        reset_offsets[1] = subspace_ptrs[1] - mit->extra_op_ptrs[0];
+                        /* use the faster adjustment */
+                        is_reset_trivial ++;
```

Comment:
Nitpick: I wouldn't put the space, though I guess it really does not matter, nor am I sure its part of any style guideline.